Kernel/Graphics/VirtIOGPU/GPU3DDevice.h

```diff
@@ -127,7 +127,7 @@ class GPU3DDevice : public CharacterDevice {
     Kernel::Graphics::VirtIOGPU::GraphicsAdapter& m_graphics_adapter;
     // Context used for kernel operations (e.g. flushing resources to scanout)
     ContextID m_kernel_context_id;
-    HashMap<RefPtr<OpenFileDescription>, RefPtr<PerContextState>> m_context_state_lookup;
+    HashMap<OpenFileDescription *, RefPtr<PerContextState>> m_context_state_lookup;
```

Comment:
Please put the * sign near the pointed object type :)
We might want to find a better solution in general to this in the future but this is good enough for now.

Reply:
Fixed.